docs/prepping-local-system-runbook-windows.md

```diff
@@ -394,4 +394,20 @@ Status: Downloaded pega-docker.downloads.pega.com/platform/installer:<version>
 
 After you push these three downloaded images to your private Docker registry, you are ready to begin deploying Pega Platform to a support Kubernetes environment. Use the runbook in this Github directory for your deployment.
 
-11. You can follow the steps similar to above when configuring `backingservices` with your deployment from steps 5 through 7 and pull `platform-services/search-n-reporting-service:<version>` instead, to configure Search and reporting service feature as an alternate search solution to `platform/search`.
+11. For Pega Infinity 8.6 and later, you can follow the below steps to configure `backingservices` and pull `platform-services/search-n-reporting-service:<version>`, to configure Search and reporting service feature as an alternate search solution to `platform/search`.
+
+12. To download your preferred version of the `search-n-reporting-service` image to your local system, specify the version tag when you enter:
```

Comment:
Pradeep -  can you please review the command for accuracy (I haven't worked with SRS image before), change to step #11, and then copy the new line into the prepping-local-system-runbook-linux.md file?